src/ui/components/Rating/index.js

```diff
@@ -50,16 +55,19 @@ export class RatingBase extends React.Component {
       const props = {
         className: classNames('Rating-choice', {
           'Rating-selected-star': rating && thisRating <= rating,
+          // Half stars are the result of averages rounded to the nearest
+          // 0.5 place; only show them when readOnly is true.
+          'Rating-half-star': (readOnly && rating && thisRating > rating &&
```

Comment:
The rating we get from the API is a `float`... if we round the rating to an integer then we can't check for half stars.